codecov.yml

```diff
@@ -0,0 +1,20 @@
+# https://docs.codecov.io/docs/codecov-yaml
```

Comment:
Is there a way to put this file somewhere other than the root of the repo?  Either in the `test` or `build` folders?  Our root directory is too cluttered the way it is.

Reply:
Unfortunately this is not possible at this time.

Reply:
@eerhardt This location is the standard convention. Deviating from it would only serve to confuse users familiar with the tools.

Reply:
It's shocking how many tools assume that they are important enough to force people to put files in the root of a repository.